scripts/link.sh

```
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# Create the dotfile symlinks
+# The home/ directory mirrors the home directory structure
+
+source "$DOTFILES_DIR/shell/utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./shell/utils.sh was not specified as input (see shellcheck -x).

(SC1091)